### PR TITLE
Matrix channel

### LINKED ISSUE #17: Add matrix Channel
Hello : 

You ask me long time ago (1 month) to add suggestion of Matrix channel in the osjs manual... but I don't understand where I have to add this link.
chat

Little summary of advantages of having matrix channel (with no drawbacks) : 
- it's a chat channel to which you don't have to register (you can be anonymous)
- You don't have to link to an existing account (privacy) : github, facebook...
- Many features like Voice or video chat, service integration (GGle calendar, Etherpad, Gdrive...) 
- Gitter is integrated in it (as you can see on the screenshot) and you can make other bridge is you want (like to a forum to have announcement of the post -- for version announcement for example)

### PR DESCRIPTION
**Pull request by frju365**
fixes #17

### CODE REVIEW

**Comment by frju365** [comment]
It's a bit weird to add it here, as moreover, nobody will have the idea to come on this page to see this. :-/

File: src/v3/resource/contrib/README.md [hunk 1]
```diff
@@ -29,3 +29,7 @@ If you want to submit your contribution, contact me by email, join the [Gitter](
 ## Filesystem
 
 * https://github.com/SpaceboyRoss01/osjs-devices-adapter - Adds all hardware devices to a root mountpoint
+
+## Community chat
```
Comment (andersevenrud) [inline src/v3/resource/contrib/README.md]: Change the header to `Community`.

File: src/v3/resource/contrib/README.md [hunk 2]
```diff
@@ -29,3 +29,7 @@ If you want to submit your contribution, contact me by email, join the [Gitter](
...
+
+* Matrix channel (use riot.im/app to join it ) : https://matrix.to/#/!lxLFDpQJHZtqePeIUK:matrix.org?via=matrix.org - Chat protocol for the person which don't want to user Gitter. Linked to Gitter official channel.
```
Comment (andersevenrud) [inline src/v3/resource/contrib/README.md]: Reformulate this sentence a bit (there's also a spelling error) to *something like maybe*:

> https://matrix.to/#/!lxLFDpQJHZtqePeIUK:matrix.org?via=matrix.org - Matrix channel linked to Gitter. Users without Github accounts can use this service as an alternative with a client like https://riot.im/app.

**Review by andersevenrud — CHANGES_REQUESTED** [review]

**Comment by andersevenrud** [comment]
> It's a bit weird to add it here, as moreover, nobody will have the idea to come on this page to see this. :-/

Maybe adding it under the contact section of the FAQ will make it easier. There's no dedicated page that links all the community services etc. (the only place where it's linked is the splash article), hence why I suggested you put it in contrib at this moment.

**Comment by frju365** [comment]
Fixed.

>Maybe adding it under the contact section of the FAQ will make it easier. There's no dedicated page that links all the community services etc. (the only place where it's linked is the splash article), hence why I suggested you put it in contrib at this moment.

Yes, indeed perhaps better. Let merge this and add it in FAQ too.

**Review by andersevenrud — APPROVED** [review]

**Pull request merged**